app/views/timeline_entries/new.html.erb

```diff
@@ -1 +1,35 @@
-<p>Placeholder</p>
+<%
+  links = [
+    {
+      text: 'Coronavirus pages',
+      href: coronavirus_pages_path
+    },
+    {
+      text: "#{@coronavirus_page.name} timeline entries",
+      href: coronavirus_page_path(slug: @coronavirus_page.slug)
+    },
+    {
+      text: "Add timeline entry"
+    },
+  ]
+
+  metadata = {
+    "Status" => "Draft",
+    "Last saved" => format_full_date_and_time(DateTime.now),
+  }
```

Comment:
It doesn't look like this metadata variable is used - unless I'm missing something?

Reply:
I think you're right. I'll remove it.